src/storage-operators/src/oneshot_source.rs

```diff
@@ -998,6 +1027,10 @@ pub enum StorageErrorXKind {
     MissingSize,
     #[error("object is missing the required '{0}' field")]
     MissingField(Arc<str>),
+    #[error("the supplied mfp filtered out a row, it is only supposed to map and project")]
```

Comment:
should we also prefix that with [internal error]? A user won't understand what an mfp is if this ever makes it to the console

Reply:
Done!